python/Ganga/GPIDev/Lib/File/LCGSEFile.py

```diff
@@ -286,18 +286,13 @@ def getWNInjectedScript(self, outputFiles, indent, patternsToZip, postProcessLoc
 
         return script
 
-    def get(self):
+    def copyTo(self, targetPath):
```

Comment:
No - as I say above, the logic for checking if a file is in the `localDir` is common to all file types and should be in the baseclass method `copyTo` which is exposed. This can then call an unexposed method `_internalCopyTo` or something that all derived classes must provide. This leaves just `get` and `copyTo` exposed (and these are the only ones that are called) and the **actual** downloading/copying code specific to a file in an internal method.